contracts/dev-utils/contracts/src/OrderTransferSimulationUtils.sol

```diff
@@ -41,6 +41,10 @@ contract OrderTransferSimulationUtils is
         TransfersSuccessful       // All transfers in the order were successful
     }
 
+    // NOTE(jalextowle): This is a random address that we use to avoid issues that addresses like `address(1)`
+    // may cause later.
+    address constant internal NONZERO = address(0x377f698C4c287018D09b516F415317aEC5919332);
```

Comment:
Nitty - this name is a bit confusing. Perhaps something that describes how it is used? Ex, `RANDOM_TAKER` or `RANDOM_ACTOR` or `UNUSED_ADDRESS`.

Reply:
Hmm, I was using `NONZERO` because the only thing that is important about this address is *that it is nonzero*. That said, I would be okay with `UNUSED`.